### PR TITLE
White screen after validation fix

### PR DESCRIPTION
**Pull request by utas-tajona**

### CODE REVIEW

File: ui/src/components/import/DataSheetView.js [hunk 1]
```diff
@@ -29,20 +29,18 @@ Object.unfreeze = function (o) {
...
 const DataSheetView = () => {
   const dispatch = useDispatch();
-  const immutableRows = useSelector((state) => state.import.rows);
-
+  const rows = useSelector((state) => state.import.rows);
   const errSelected = useSelector((state) => state.import.errSelected);
   const [gridApi, setGridApi] = useState(null);
-
-  var rows = immutableRows.map(Object.unfreeze);
+  // var rows = immutableRows.map(Object.unfreeze);
```
Comment (nspool) [inline ui/src/components/import/DataSheetView.js:35]: Better to remove than comment out I think.

File: ui/src/components/import/DataSheetView.js [hunk 2]
```diff
@@ -106,9 +113,9 @@ const DataSheetView = () => {
             onCellValueChanged={onCellChanged}
             columnDefs={colDefinition}
             groupDefaultExpanded={4}
-            rowData={rows}
             rowHeight={18}
             animateRows={true}
+            //  rowData={rows}
```
Comment (nspool) [inline ui/src/components/import/DataSheetView.js:118]: can be removed.

File: ui/src/components/import/middleware/create-import.js [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
-        yield put(ImportFailed([e]));
+  try {
+    const {response} = yield call(submitJobFile, params.payload);
+    const data = response.data || {};
+    var errors = data.errors || [];
+    if (data.error) {
+      errors = [{message: data.error}];
     }
+    console.debug('data', data);
+    console.debug('errors', errors);
```
Comment (nspool) [inline ui/src/components/import/middleware/create-import.js:18]: Should remove these debug statements.

**Review by nspool — CHANGES_REQUESTED** [review]
Looks good. Just needs some extra lines removed.

**Review by jonescc — COMMENTED** [review]
Have tested this out locally and validation looks to work better now (no white page), can copy/paste and depth has been added to pivotting

**Comment by jonescc** [comment]
@nspool?

**Review by nspool — APPROVED** [review]

**Pull request merged**